bin/node-template/node/src/command.rs

```diff
@@ -21,32 +21,32 @@ use crate::service;
 use sc_cli::{SubstrateCli, RuntimeVersion, Role, ChainSpec};
 
 impl SubstrateCli for Cli {
-	fn impl_name() -> &'static str {
-		"Substrate Node"
+	fn impl_name() -> String {
+		String::from("Substrate Node")
```

Comment:
```suggestion
		"Substrate Node".into()
```

Here and everywhere else please

Reply:
gonna change that on this pr and polkadot's and cumulus :)

Reply:
wasn't sure what the best way to do that was